submissions/korotushak96/js-dom/index.css

```diff
@@ -0,0 +1,142 @@
+@media screen {
+    body{
+        box-sizing: border-box;
+    }
+    .burger {
+        position: absolute;
+        top: 17px;
+        right: 10px;
+        cursor: pointer;
+    }
+
+    .bar1, .bar2, .bar3 {
+        width: 35px;
+        height: 5px;
+        background-color: #333;
+        margin: 6px 0;
+        transition: 0.4s;
+    }
+
+    .change .bar1 {
+        transform: rotate(-45deg) translate(-10px, 5px);
+    }
+
+    .change .bar2 {
+        opacity: 0;
+    }
+
+    .change .bar3 {
+        transform: rotate(45deg)  translate(-11px, -5px);
+    }
+
+    .container{
+        position: relative;
+    }
+
+    .menu{
+        position: absolute;
+        font-size: 30px;
+        color: blue;
+        transform: translate(-300px, 0px);
+        transition: 0.5s;
+    }
+
+    .menu.open{
+        transform: translate(0px, 0px);
+    }
+
+    .menu-item{
+        background-color: black;
+        padding-bottom: 5px;
+        margin-bottom: 0px;
+        border-bottom: 2px solid lightblue;
+        cursor: pointer;
+    }
+
+    .menu-item:hover{
+        background-color: rgba(0, 0, 0, 0.623)
+    }
+
+    .chosen{
+        background-color: rgba(78, 100, 110, 0.623)
+    }
+
+    .content-item{
+        display: none;
+        font-size: 20px;
+        background-color: rgb(83, 54, 54);
+        color: rgb(20, 102, 197)
+    }
+
+    .content-item_image{
+        width: 100%;
+    }
+
+    .content-item.active{
+        display: block;
+    } 
+}
+@media screen and (min-width: 421px) and (max-width: 830px){
+    .content-item_image{
+        width: 75vh;
+        height: auto;
+        margin-left: 5%;
+        margin-right: 5%;
+    }
+    .content-item_text{
+        margin-left: 5%;
+        margin-right: 5%;
+    }
+}
+@media screen and (min-width: 831px) and (max-width: 1199px){
```

Comment:
And in general, try to use colour [palettes](https://flatuicolors.com/) in order to combine different colours from one palette.
It will make your app more pleasant and simple in perception for users.

Reply:
Thanks. I've never heard about this before. It's really great resource!